doc/user-guides/simple-rl-for-app-developers.md

```diff
@@ -102,10 +102,13 @@ curl -H 'Host: api.toystore.com' http://$GATEWAY_URL/toys -i
 # HTTP/1.1 200 OK
 ```
 
-> **Note**: If the command above fails to hit the Toy Store API on your environment, try forwarding requests to the service:
+> **Note**: If the command above fails to hit the Toy Store API on your environment, try forwarding requests to the service and accessing over localhost:
 >
 > ```sh
 > kubectl port-forward -n istio-system service/istio-ingressgateway 9080:80 2>&1 >/dev/null &
+> export GATEWAY_URL=localhost:9080
+> curl -H 'Host: api.toystore.com' http://$GATEWAY_URL/toys -i
+> # HTTP/1.1 200 OK
```

Comment:
```suggestion
> kubectl port-forward -n istio-system service/istio-ingressgateway-istio 9080:80 2>&1 >/dev/null &
> export GATEWAY_URL=localhost:9080
> ```
> ```sh
> curl -H 'Host: api.toystore.com' http://$GATEWAY_URL/toys -i
> # HTTP/1.1 200 OK
```

1. Use `istio-ingressgateway-istio` service
2. There seems to be a delay with the port forward, running the commands in one block give this error on `curl`:
```sh
[1] 74443
curl: (56) Recv failure: Connection reset by peer
```
but re-running the curl command works, so just suggesting to break up the commands into seperate blocks to help with this